openshift/catalogd.Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.23-openshift-4.19 AS builder
```

Comment:
I guess this is 1.23.2 version of the go.

Reply:
This file is the same merged at https://github.com/openshift/operator-framework-catalogd and passing in the CI/prow jobs there. See; https://github.com/openshift/operator-framework-catalogd/blob/main/openshift/Dockerfile